ethexe/consensus/src/validator/core.rs

```diff
@@ -282,6 +289,16 @@ impl ValidatorCore {
             Ok(digest)
         }
     }
+
+    pub fn process_injected_transaction(&mut self, tx: SignedInjectedTransaction) -> Result<()> {
+        // TODO kuzmindev : This check in future should be removed, because of by design main service [`ethexe-service::Service`]
```

Comment:
It seems that this check could be removed already. We basically dont care if somehow received injected tx not for us if we're able to insert it